lib/src/crawl_async.dart

```diff
@@ -29,6 +29,10 @@ final _empty = Future<Null>.value(null);
 /// If either [readNode] or [edges] throws the error will be forwarded
 /// through the result stream and no further nodes will be crawled, though some
 /// work may have already been started.
+///
+/// Crawling is eager, calls to [edges] may overlap with other calls that have
```

Comment:
Weird sentence. Missing `so`? `Crawling is eager, so calls to`

Reply:
Done